lib/node-detector.js

```diff
@@ -71,7 +72,7 @@ class NodeDetector {
       log.debug(err);
       return null;
     }
-    let nodePath = stdout.replace("\n", "");
+    let nodePath = stdout.split(/\r?\n/)[0];
```

Comment:
it probably makes sense to extract path extraction into a separate helper function

Reply:
This line was already filtered as macOS
https://github.com/appium/appium-doctor/pull/45/files#diff-5cee4c29bc65633956405e495b2a9f4aR44